storage/pebble/operation/prefix.go

```diff
@@ -11,9 +11,6 @@ const (
 	//lint:ignore U1000 Ignore unused variable warning
 	// job queue consumers and producers
 	codeJobConsumerProcessed = 70
-
-	// legacy codes (should be cleaned up)
-	codeChunkDataPack = 100
```

Comment:
should this code be reserved for some period of time?

Reply:
If reserving is to reverting the changes, I would revert by reverting the whole feature branch, rather than doing at the code level. 

So I think it's OK to just clean up